static/js/components/Paginator.js

```diff
@@ -8,22 +8,17 @@ class Paginator extends React.Component<*, void> {
     const { currentPage, totalPages } = this.props
     return (
       <div className="paginator">
-        <span className="paginator-current-range">
-          Page <span className="paginator-current-page">{currentPage}</span>
-          &nbsp;of&nbsp;
-          <span className="paginator-total-pages">{totalPages}</span>
-        </span>
-      
-        <span className="paginator-buttons"
-          style={{
-            marginLeft: '.5em',
-            position:   'relative',
-            top:        '-.1em',
-          }}
-        >
+        <div className="contents" style={{position: "relative"}}>
```

Comment:
Minor nitpick: maybe `position: relative` could be added to a new style class, and that class could be added here, instead of using `style`?  `<div className="contents contents_relative">`

Reply:
Let's keep the `style` here for now. I'll go ahead and merge.